### ISSUE TITLE
UCP: make configuration of lock manager readable

### ISSUE THREAD

**Issue by youjiali1995** [tikv/tikv#7143]
## Description
The `wait-for-lock-timeout` and `wake-up-delay-duration` are pure numbers and in milliseconds. They're not readable:
```
wait-for-lock-timeout = 1000
wake-up-delay-duration = 20
```

Use `ReadableDuration` instead:
https://github.com/tikv/tikv/blob/250258cb239ab0e2554bcec53967523552905ae8/components/tikv_util/src/config.rs#L244

These configurations are being used in v3.x. We should guarantee backward compatibility. Don't forget to file a corresponding PR to https://github.com/pingcap/tidb-ansible.

## Score
* 100

## Mentor(s)
* @youjiali1995  

## Recommended Skills
* Rust programming

**Comment by zhangjinpeng1987** [comment]
@youjiali1995 can you add this issue to https://github.com/tikv/tikv/projects/20 and let contributor do it?

**Comment by youjiali1995** [comment]
> @youjiali1995 can you add this issue to https://github.com/tikv/tikv/projects/20 and let contributor do it?

OK. I will.

**Comment by SSebo** [comment]
@youjiali1995 mentor, PTAL, I give up because I have another issue to working on.

**Comment by SSebo** [comment]
@you06 Sorry for bother you again, I give up this issue after I make a PR, Could I get the points ?

**Comment by youjiali1995** [comment]
> @youjiali1995 mentor, I make a PR, PTAL

Okay

**Comment by you06** [comment]
@SSebo You're doing the right way, except you don't need to give up the issue after making a PR, and your give up does not take effect. The points will be rewarded after #7197 got merged. Good luck!
